pages/favorites.js

```diff
@@ -21,15 +28,40 @@ export default function FavoriteCountriesPage({
     listFavoriteCountries.find((info) => info.name === country.name.common)
   );
 
+  function handleAddEntry(newEntry) {
+    setEntries([{ ...newEntry }, ...entries]);
```

Comment:
I think the spread operator for ...newEntry is not needed since newEntry is already an object

Reply:
Hi Nina, is actually doesn't work without the operator. So I kept it :)